<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Storage_box</class>
 <widget class="QMainWindow" name="Storage_box">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>191</width>
      <height>44</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 30pt &quot;배달의민족 한나체 Pro&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;택배 보관함&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_17">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_18">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_19">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;01&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;02&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;04&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;03&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_8">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;08&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;07&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;06&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;05&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;09&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_11">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;11&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;12&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_15">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;15&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;13&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;14&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_16">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;16&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_17">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;17&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_18">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;18&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_19">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;19&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 9px;
border-color : rgb(68, 114, 196);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;사용가능&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 255, 255);
border-style : outset;
border-width : 3px;
border-radius : 9px;
border-color : rgb(68, 114, 196);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;사용중&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushB3423n_20">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>111</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 180, 180);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_20">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>60</y>
      <width>111</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(68, 114, 196);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(129, 143, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_20">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;20&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushB3423n_21">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(230, 230, 230);
border-style : outset;
border-width : 3px;
border-radius : 5px;
border-color : rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_21">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;
background-color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;무인 택배함&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Index_label_22">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>230</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;
background-color: rgb(180, 180, 180);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Made in&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;박경은&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Guide_label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;배달의민족 주아&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;★ 맞기실 택배의 보관함을 선택하세요 ★&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>pushButton_12</zorder>
   <zorder>pushButton_13</zorder>
   <zorder>pushButton_14</zorder>
   <zorder>pushButton_15</zorder>
   <zorder>pushButton_16</zorder>
   <zorder>pushButton_17</zorder>
   <zorder>pushButton_18</zorder>
   <zorder>pushButton_19</zorder>
   <zorder>Index_label_1</zorder>
   <zorder>Index_label_2</zorder>
   <zorder>Index_label_4</zorder>
   <zorder>Index_label_3</zorder>
   <zorder>Index_label_8</zorder>
   <zorder>Index_label_7</zorder>
   <zorder>Index_label_6</zorder>
   <zorder>Index_label_5</zorder>
   <zorder>Index_label_9</zorder>
   <zorder>Index_label_11</zorder>
   <zorder>Index_label_12</zorder>
   <zorder>Index_label_10</zorder>
   <zorder>Index_label_15</zorder>
   <zorder>Index_label_13</zorder>
   <zorder>Index_label_14</zorder>
   <zorder>Index_label_16</zorder>
   <zorder>Index_label_17</zorder>
   <zorder>Index_label_18</zorder>
   <zorder>Index_label_19</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>pushB3423n_20</zorder>
   <zorder>pushButton_20</zorder>
   <zorder>Index_label_20</zorder>
   <zorder>pushB3423n_21</zorder>
   <zorder>Index_label_21</zorder>
   <zorder>Index_label_22</zorder>
   <zorder>Guide_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
